### PR TITLE
Rejuvenating cyborg give charges

### PR DESCRIPTION
**Pull request by Saicchi**
## About the Pull Request

Add recharge for Robots Rejuvenate Command

## Why It's Good For The Game

Makes rejuvenating a cyborg with no charges get some.

## Changelog

:cl:
fix: Rejuvenating robots now gives them charge
/:cl:

### CODE REVIEW

**Comment by 77seven77seven** [comment]
a small but epic change

File: code/modules/mob/living/silicon/robot/robot_damage.dm [hunk 1]
```diff
@@ -32,6 +32,11 @@
 	else
 		heal_overall_damage(0, -amount)
 
+/mob/living/silicon/robot/rejuvenate()
+	..()
+	if(src.cell)
+		src.cell.give(src.cell.maxcharge)
```
Comment (EgorDinamit) [inline code/modules/mob/living/silicon/robot/robot_damage.dm]: `src` is obsolete here.
```suggestion
	if(cell)
		cell.give(cell.maxcharge)
```

Comment (Saicchi) [reply code/modules/mob/living/silicon/robot/robot_damage.dm]: I'm a big fan of 'Explicit is better than implicit.' I'd rather not remove it unless it clashes with the code guidelines.

Comment (EgorDinamit) [reply code/modules/mob/living/silicon/robot/robot_damage.dm]: I am leaving the suggestion here for a reason. There is 0 reasons to utilize `src.var` outside of situations where you have a proc's argument named the same way as a variable of src object (i.e. `src.thing = thing`).

Comment (Tsurupeta) [reply code/modules/mob/living/silicon/robot/robot_damage.dm]: Just remove it

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**